content/Assets/Styles/index.scss

```diff
@@ -2,23 +2,24 @@
    #Etch.OrchardCore.ThemeBoilerplate THEME
    ========================================================================== */
 
-@import "variables/all";
+@import 'variables/all';
```

Comment:
How come all these changed to single quotes? Assume it was probably prettier, which is fine, means we have to change all the others. Wouldn't do that now, we can do it to all the Sass files as well as any other prettier conventions.

Reply:
It's prettier, yep.